astropy/table/row.py

```diff
@@ -131,11 +131,11 @@ def get(self, key, default=None, /):
         --------
         >>> from astropy.table import Table
         >>> t = Table({"a": [2, 3, 5], "b": [7, 11, 13]})
-        >>> t[0].get("a")
-        np.int64(2)
-        >>> t[1].get("b", 0)
-        np.int64(11)
-        >>> t[2].get("c", 0)
+        >>> int(t[0].get("a"))
```

Comment:
Not sure if we want to tell people to always recast to int. What about ellipsis?

Reply:
Just my opinion, but eliding the output feels more harmful than casting, readability wise.

Reply:
Depends on the purpose of the documentation. The first priority is we send the right message in the doc example. So I'll defer approval to `table` maintainers. Thanks for your patience!